monitoring/bin/deploy_monitoring_openshift.sh

```diff
@@ -52,11 +52,11 @@ else
 fi
 
 # Elasticsearch Datasource for Grafana
-ELASTICSEARCH_DATASOURCE="${ELASTICSEARCH_DATASOURCE:-false}"
-if [ "$ELASTICSEARCH_DATASOURCE" == "true" ]; then
+LOGGING_DATASOURCE="${LOGGING_DATASOURCE:-false}"
+if [ "$LOGGING_DATASOURCE" == "true" ]; then
   configureElasticsearchDatasource
```

Comment:
Does the configureElasticsearchDatasource function still exist?  I thought it was replaced by the create_logging_datasource.sh script?

Reply:
It does not exist - it has been correctly updated to point to the create_logging_datasource script.